src/app/dashboard/accounts/page.tsx

```diff
@@ -28,26 +51,94 @@ export default function AccountsPage(): JSX.Element {
   accounts = accounts || [];
   todayPrices = todayPrices || new PriceDBMap();
 
+  let tree: AccountsTree = {
+    account: {},
+    total: new Money(0, 'EUR'),
+    children: [],
+  };
+  const root = accounts.find(a => a.type === 'ROOT');
+  if (root && !todayPrices.isEmpty) {
+    tree = buildNestedRows(root, accounts, todayPrices, dateRange);
+  }
```

Comment:
We build the tree in the page rather than in the AccountsTable because the tree will be needed for rendering other components in the future version of the page.